packages/pluggableWidgets/slider-native/src/Slider.tsx

```diff
@@ -47,11 +53,13 @@ export function Slider(props: Props): ReactElement {
             }
 
             lastValue.current = values[0];
-            props.valueAttribute.setValue(new Big(values[0]));
+            if (props.stepSize.status === ValueStatus.Available) {
+                props.valueAttribute.setValue(new Big(values[0].toFixed(decimalCount(props.stepSize.value))));
+            }
 
             executeAction(props.onChange);
         },
-        [lastValue.current, props.valueAttribute, props.onChange]
+        [props.valueAttribute, props.onChange, props.stepSize, decimalCount]
```

Comment:
Howcome you dropped the `lastValue.current` from the deps list?

Reply:
Ah I meant to only remove `current`.  Having `lastValue.current` as a dependency of useCallback is not valid as mutating it doesn't re-render the component. But `lastValue` should still be there so I re-added it :)